internal/dispatch/remote/cluster.go

```diff
@@ -55,10 +55,13 @@ var primaryDispatch = prometheus.NewCounterVec(prometheus.CounterOpts{
 
 // defaultStartingPrimaryHedgingDelay is the delay used by default for primary calls (when secondaries are available),
 // before statistics are available to determine the actual delay.
-const defaultStartingPrimaryHedgingDelay = 5 * time.Millisecond
+const defaultStartingPrimaryHedgingDelay = 1 * time.Millisecond
+
+// maximumHedgingDelay is the maximum delay used for hedging requests to the primary dispatcher.
+const maximumHedgingDelay = 5 * time.Millisecond
```

Comment:
I'd suggest that you make this configurable. Under certain load conditions this may not be enough

Reply:
The concern is that if we're going much beyond this, then the overhead for normal calls will be too large vs the "extra work" we're doing

Reply:
I think it's entirely context dependent. We could have a data store dispatches in average in 10ms, and a secondary that does so in 7ms. You still get net gains out of dispatching to the secondary, but the cap won't allow it.

Reply:
Remember - this is the delay from the *secondary* dispatch, which means it should never be more than a few ms max; if it is more than that, then we don't want to delay the primary anyway, since the secondary is being overwhelmingly slow